packages/react/src/components/Tile/Tile.mdx

```diff
@@ -52,6 +52,32 @@ import { Story, Props, Source, Preview } from '@storybook/addon-docs';
   <Story id="components-tile--radio" />
 </Preview>
 
+## Experimental Improved Contrast
+
+In Sass, you can enable the improved contrast tiles via a feature flag.
+
+```scss
+@include feature-flags.enable('enable-experimental-tile-contrast');
+```
+
```

Comment:
@tay1orjones is this the correct way to do this? I wasn't sure.

Reply:
Yeah I think that should work. The other way I had working for testing was

```
@use '@carbon/react/scss/feature-flags' with (
  $feature-flags: (
    'enable-experimental-tile-contrast': true,
  )
);
```